app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -30,6 +31,8 @@ data class ValidatedIntentData(val url: String, val source: Session.Source)
 object IntentValidator {
     @VisibleForTesting const val DIAL_PARAMS_KEY = "com.amazon.extra.DIAL_PARAM"
     private const val ACTIVE_EXPERIMENTS_KEY = "activeExperiments"
+    private const val IS_QA_BUILD_KEY = "isQA"
+    private const val FETCH_DELAY_KEY = "fetchDelay"
```

Comment:
nit: EXTRA_FETCH_DELAY or KEY_EXTRA_FETCH_DELAY -> it's helpful to prepend KEY or EXTRA so that it's really clear which constants are intended to be used where, if they're used in a consistent way.